databroker/v1.py

```diff
@@ -1568,7 +1568,7 @@ def _pretty_print_time(timestamp):
     return '{ago} ({date})'.format(ago=ago, date=dt)
 
 
-def from_config(config, auto_register=True, name=None):
+def from_config(config, auto_register=True, name=None, transforms=None):
```

Comment:
To be consistent with how `handler_registry` is handled, and to avoid making things too complex, I don't think we should allow the user to specify `transforms` here. They should just come `config`.

Reply:
updated